sources/osgViewer/Viewer.js

```diff
@@ -404,18 +408,12 @@ Viewer.prototype = MACROUTILS.objectInherit( View.prototype, {
         var self = this;
         var render = function () {
             if ( !self.done() ) {
-
-                var isPresentingVR = self._hmd && self._hmd.isPresenting;
-
-                if ( isPresentingVR )
+                if ( self._hmd && self._hmd.isPresenting )
```

Comment:
> It's just that I don't feel comfortable doing an extra conditional every frame if it is really not needed.

^ that's really nothing, it's not even a function, no dynamic lookup or math, no allocation, just check the visitor/rendering code in osgjs and will see that it's literally nothing in comparison of what is already done.

Take for example the begin/end Frame that does many many useless tests... https://github.com/cedricpinson/osgjs/blob/develop/sources/osgViewer/Viewer.js#L326-Lundefined

Anyway I updated the PR, as long as the 2nd parameter doesn't hinder the vr function it's fine by me